lib/_http_server.js

```diff
@@ -477,20 +478,25 @@ function connectionListener(socket) {
       // if the user never called req.read(), and didn't pipe() or
       // .resume() or .on('data'), then we call req._dump() so that the
       // bytes will be pulled off the wire.
-      if (!req._consuming && !req._readableState.resumeScheduled)
+      if (!req._consuming && !req._readableState.resumeScheduled) {
         req._dump();
+      }
 
       res.detachSocket(socket);
+      res._last = outgoing.length === 0;
```

Comment:
In debugging I found that when res was last in ```outgoing```, and ```_last``` flag is not correct, but we need it calculated correctly for next lenes, to destroy socket if it is last.